src/docs/sphinx/t_conduit_docs_tutorial_python_relay_mpi_examples_out.txt

```diff
@@ -9,102 +9,102 @@ Updating test list for fixtures
-116: test_004_mpi_bcast_using_schema (t_conduit_docs_tutorial_python_relay_mpi_examples.Conduit_Tutorial_Python_Relay_IO_Handle) ... ok
-116: test_005_mpi_sum_all_reduce (t_conduit_docs_tutorial_python_relay_mpi_examples.Conduit_Tutorial_Python_Relay_IO_Handle) ... ok
-116: test_005_mpi_sum_all_reduce (t_conduit_docs_tutorial_python_relay_mpi_examples.Conduit_Tutorial_Python_Relay_IO_Handle) ... ok
-116: test_006_mpi_all_gather (t_conduit_docs_tutorial_python_relay_mpi_examples.Conduit_Tutorial_Python_Relay_IO_Handle) ... ok
-116: test_006_mpi_all_gather (t_conduit_docs_tutorial_python_relay_mpi_examples.Conduit_Tutorial_Python_Relay_IO_Handle) ... ok
-116: 
-116: ----------------------------------------------------------------------
-116: Ran 6 tests in 0.274s
-116: 
-116: OK
-116: ok
-116: 
-116: ----------------------------------------------------------------------
-116: Ran 6 tests in 0.274s
-116: 
-116: OK
-116: 
-116: BEGIN_EXAMPLE("py_mpi_send_and_recv_using_schema")
-116: [rank: 0] sending: 
-116: a: 
-116:   data: 1.0
-116:   more_data: 2.0
-116:   b: 
-116:     my_string: "value"
-116: 
-116: 
-116: END_EXAMPLE("py_mpi_send_and_recv_using_schema")
-116: 
-116: BEGIN_EXAMPLE("py_mpi_send_and_recv")
-116: [rank: 0] sending: [0, 1, 4, 9]
-116: 
-116: END_EXAMPLE("py_mpi_send_and_recv")
-116: 
-116: BEGIN_EXAMPLE("py_mpi_bcast_using_schema")
-116: [rank: 0] broadcasting: 
-116: a: 
-116:   data: 1.0
-116:   more_data: 2.0
-116:   b: 
-116:     my_string: "value"
-116: 
-116: 
-116: END_EXAMPLE("py_mpi_bcast_using_schema")
-116: 
-116: BEGIN_EXAMPLE("py_mpi_bcast")
-116: [rank: 0] broadcasting: [0, 1, 4, 9]
-116: 
-116: END_EXAMPLE("py_mpi_bcast")
-116: 
-116: BEGIN_EXAMPLE("py_mpi_sum_all_reduce")
-116: [rank: 0] sum reduce result: [2, 2, 2, 2]
-116: 
-116: END_EXAMPLE("py_mpi_sum_all_reduce")
-116: 
-116: BEGIN_EXAMPLE("py_mpi_all_gather_using_schema")
-116: [rank: 0] all gather using schema result: 
-116: - [0, 0, 0, 0]
-116: - [1, 1, 1, 1]
-116: 
-116: 
-116: END_EXAMPLE("py_mpi_all_gather_using_schema")
-116: [rank: 1] received: 
-116: a: 
-116:   data: 1.0
-116:   more_data: 2.0
-116:   b: 
-116:     my_string: "value"
-116: 
-116: [rank: 1] received: [0, 1, 4, 9]
-116: [rank: 1] received: 
-116: a: 
-116:   data: 1.0
-116:   more_data: 2.0
-116:   b: 
-116:     my_string: "value"
-116: 
-116: [rank: 1] received: [0, 1, 4, 9]
-1/1 Test #116: t_conduit_docs_tutorial_python_relay_mpi_examples ...   Passed    0.48 sec
+118: Test command: /Users/harrison37/Work/github/llnl/conduit/uberenv_libs/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.0-apple/mpich-3.3.1-kkqsshrtda4dgcnbbwxndombmgfsmp4f/bin/mpiexec "-n" "2" "/Users/harrison37/Work/github/llnl/conduit/uberenv_libs/spack/opt/spack/darwin-mojave-x86_64/clang-10.0.0-apple/python-3.7.6-zd24bom2hns4o42jlzdptijz54ogai7c/bin/python3.7" "-B" "-m" "unittest" "-v" "t_conduit_docs_tutorial_python_relay_mpi_examples"
```

Comment:
Same for all of the rest of these. I don't know what the best solution is here; can we just omit the line number from the output?

Reply:
I agree there is a lot of noise here. But those parts of the files aren't pulled into the docs, where other parts are automatically pulled in (via tags).  It is a bit annoying to skip as part of the review, but trying to avoid the changes in the first place would be hard, i worry that creating system to do so could cause surprises.

Reply:
Could we just trim all non-tagged segments of these files, or is there something important we want to retain in the interim segments?

Reply:
I am sure we could create a solution. But there are bound to be corner cases, they scare me. 

Adding the script to regen these was a huge improvement and lowered the barrier big time. Any clean up could be processed by that script.  (scripts/regen_docs_outputs.py)

Reply:
Added this issue so we can continue to discuss: 
https://github.com/LLNL/conduit/issues/620